community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -6,6 +6,7 @@ The OpenJS Foundation’s community fund supports travel and attendance at event
 
 * Collaborators
 * Standards body participants
+* Over the age of 18
```

Comment:
```suggestion
* At least 18 years old and recognized as adults in the destination country, based on local laws (ref: [Age of Majority](https://en.wikipedia.org/wiki/Age_of_majority))
```

Maybe is safer to ensure that the requester is at least 18 (US Compliance) and also that can be considered "adult" in the destination country, that way we reduce the responsibilities a lot.

I can see some edge cases in the list (Canada +19, Puerto Rico +21, US (Nebraska) +19...)